src/components/feedback/Toast/useToast.ts

```diff
@@ -39,9 +39,9 @@ const useToast = () => {
         [T, always(Slide)]
       ])
 
-      const options: ToastifyToastOptions = {
+      const options: ToastifyToastOptionsCustom = {
         ...restOptions,
-        autoClose: autoClose || false,
+        autoClose: autoClose,
```

Comment:
is the same name. just `autoClose` should be enough

Reply:
modified it accordingly in the latest commit